automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookHailSpec.scala

```diff
@@ -13,19 +13,10 @@ import scala.language.postfixOps
 class NotebookHailSpec extends ClusterFixtureSpec with NotebookTestUtils {
 
   "Leonardo notebooks" - {
-
-    val hailUploadFile = ResourceFile(s"diff-tests/import-hail.ipynb")
+    // Should match the HAILHASH env var in the Jupyter Dockerfile
+    val expectedHailVersion = "0.2.11-bb41118d9587"
     val hailTutorialUploadFile = ResourceFile(s"diff-tests/hail-tutorial.ipynb")
 
-    "should upload notebook and verify execution" taggedAs Tags.SmokeTest in { clusterFixture =>
```

Comment:
I removed this test because it wasn't actually doing anything. :( It was uploading a notebook which imports hail but not verifying the result. I replaced it with a test which imports hail and checks the version.